### PR TITLE
HCK15

### PR DESCRIPTION
**Pull request by bendichter**
Add main page and draft agenda for NWB User and Developer Days 2023 at Janelia

### CODE REVIEW

**Comment by bendichter** [comment]
@CodyCBakerPhD the website is not rendering here. I'm not sure why

**Comment by bendichter** [comment]
@CodyCBakerPhD it's working now! It takes about 7 minutes after the link is posted to actually work

**Comment by CodyCBakerPhD** [comment]
@bendichter It's because the page itself is still building (you can always check https://github.com/NeurodataWithoutBorders/nwb_hackathons/actions to see)

To view a preview on a PR, you'll have to wait ~10 minutes after the most recently merged PR (which is what triggers a rebuild)

You may have to retrigger the preview after that time as well, not sure

File: HCK14_2023_Janelia/README.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
+# NWB User Days and Developer Hackathon 2023
```
Comment (oruebel) [inline HCK14_2023_Janelia/README.md:1]: Can you please rename the folder to ``HCK15_2023_Janelia`` (i.e., 15 instead of 14)

**Comment by bendichter** [comment]
https://neurodatawithoutborders.github.io/nwb_hackathons/pr-preview/pr-204/HCK15_2023_Janelia/

File: HCK14_2023_Janelia/README.md (hunk 1 above)
Comment (oruebel) [reply HCK14_2023_Janelia/README.md]: I just made that change. I'll add a few more edits now and then I think this is good to merge.

**Comment by bendichter** [comment]
It might make more sense to have the code of conduct on a standalone page and then link to it

**Comment by rly** [comment]
> It might make more sense to have the code of conduct on a standalone page and then link to it

That makes sense.

**Review by oruebel — APPROVED** [review]
I'm done with my changes for now. Looks good to me.

File: HCK15_2023_Janelia/README.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+data and metadata. NWB enables data sharing and reuse and reduces the energy barrier to applying data analytics both within
+and across labs. NWB has seen wide adoption in the neurophysiology community, and there are now over 100 datasets on the
+DANDI Archive in NWB, including data from the Allen Institute and the International Brain Laboratory.
+
+The **User Days** will train users how to convert their data to NWB and publish it on the DANDI Archive. 
+We will work with members of the neuroscience community that want to apply NWB to their datasets. We will train 
+attendees, starting from the basics and proceeding to advanced data engineering techniques to maximally utilize the
+features of the HDF5 and Zarr backends. Attendees will also be trained in the creation of NWB extensions.
+
+<<<<<<< HEAD:HCK15_16_2023_Janelia/README.md
```
Comment (bendichter) [inline HCK15_2023_Janelia/README.md:89]:
```suggestion
```

Comment (oruebel) [reply HCK15_2023_Janelia/README.md:89]: Oops, that was probably my fault. I ran into merge conflicts and probably forgot to remove that line. Thanks for the fix.

**Pull request merged**